autofit/non_linear/mcmc/emcee/emcee.py

```diff
@@ -71,6 +71,8 @@ def __init__(
             An SQLalchemy session instance so the results of the model-fit are written to an SQLite database.
         """
 
+        number_of_cores = number_of_cores or self._config("parallel", "number_of_cores")

```

Comment:
This is inconsistent with the other number_of_cores checks which explicitly check for None